runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -242,16 +242,21 @@ public void onGameTick(GameTick tick)
 			if (!Objects.equals(lastArenaTicketPosition, newTicketPosition))
 			{
 				// We don't want to notify when players first enter the course
-				if (lastArenaTicketPosition != null && newTicketPosition != null)
+				if (lastArenaTicketPosition != null)
 				{
-					if (config.notifyAgilityArena())
+					// We want to ignore the plane so that the timer won't reset when falling and climbing back up
+					if (!(lastArenaTicketPosition.getX() == newTicketPosition.getX()
```

Comment:
How does the hint arrow work in agility arena when you fall? It moves it down a plane? And I don't think it is safe to assume it is never null.

Reply:
Yeah it moves to the plane 0, as far as I'm avare it's never null when the player is in the arena but I can add the check back